make bootstrap: yarn not found

Following up from my previous issue #302, I now manage to pass all steps but fail due to a missing font module:
```
Step 25/25 : CMD ["gunicorn", "-c", "/usr/local/etc/gunicorn/pixel.py", "pixel.wsgi:application"]
 ---> Using cache
 ---> 3ecf178890e1
Successfully built 3ecf178890e1
Successfully tagged pixel-dev_web:latest
Waiting until database is up
make[1]: Entering directory '/home/lgatto/tmp/pixel'
Starting pixel-dev_db_1 ... done
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, core, data, sessions, submission, viewflow
Running migrations:
  No migrations to apply.
make[1]: Leaving directory '/home/lgatto/tmp/pixel'
Starting pixel-dev_db_1 ... done
Installed 6 object(s) from 1 fixture(s)
Starting pixel-dev_db_1 ... done
Installed 10 object(s) from 1 fixture(s)
Starting pixel-dev_db_1 ... done
Will generate 3 experiments and 10 pixel sets with 20 pixels each…
Successfully generated fixtures
/bin/sh: line 1: yarn: command not found
make[1]: Entering directory '/home/lgatto/tmp/pixel'
cp: cannot stat 'node_modules/font-awesome/fonts/*': No such file or directory
make[1]: *** [Makefile:65: build-css] Error 1
make[1]: Leaving directory '/home/lgatto/tmp/pixel'
make: *** [Makefile:45: bootstrap] Error 2
```
I had a look at #59, which seems related - I do not have a `node_modules` directory in the `pixel` base dir and I haven't done any installation of `yarn` nor `npm` before.

Next I ran 
```
$ docker-compose -p pixel-dev -f docker-compose.yml run --rm node yarn install
```
giving
```
Pulling node (node:8)...
8: Pulling from library/node
61be48634cb9: Pull complete
fa696905a590: Pull complete
b6dd2322bbef: Pull complete
32477089adb4: Pull complete
febe7209ec28: Pull complete
4364cbe57162: Pull complete
ace5c680ff94: Pull complete
4acd6a9b7a48: Pull complete
yarn install v1.9.4
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.1.2: The platform "linux" is incompatible with this module.
info "fsevents@1.1.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "1.12.3", while you're on "1.9.4".
info To upgrade, run the following command:
$ curl --compressed -o- -L https://yarnpkg.com/install.sh | bash
Done in 10.40s.
```
Now I successfully install the missing fonts and have the missing directory, but
```
$ make bootsrap
[...]
Step 25/25 : CMD ["gunicorn", "-c", "/usr/local/etc/gunicorn/pixel.py", "pixel.wsgi:application"]
 ---> Using cache
 ---> 3ecf178890e1
Successfully built 3ecf178890e1
Successfully tagged pixel-dev_web:latest
Waiting until database is up
make[1]: Entering directory '/home/lgatto/tmp/pixel'
Starting pixel-dev_db_1 ... done
Operations to perform:
  Apply all migrations: admin, auth, contenttypes, core, data, sessions, submission, viewflow
Running migrations:
  No migrations to apply.
make[1]: Leaving directory '/home/lgatto/tmp/pixel'
Starting pixel-dev_db_1 ... done
Installed 6 object(s) from 1 fixture(s)
Starting pixel-dev_db_1 ... done
Installed 10 object(s) from 1 fixture(s)
Starting pixel-dev_db_1 ... done
Will generate 3 experiments and 10 pixel sets with 20 pixels each…
Successfully generated fixtures
/bin/sh: line 1: yarn: command not found
make[1]: Entering directory '/home/lgatto/tmp/pixel'
make[1]: yarn: Command not found
make[1]: *** [Makefile:66: build-css] Error 127
make[1]: Leaving directory '/home/lgatto/tmp/pixel'
make: *** [Makefile:45: bootstrap] Error 2
```
Not sure where to go from now.

Thank you in advance for your help.

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,21 +11,21 @@ ARG BUILD_DATE
 ARG VCS_REF
 ARG VERSION
 LABEL org.label-schema.build-date=$BUILD_DATE \
-      org.label-schema.name="Pixel" \
-      org.label-schema.description="Integration of smart 'omics' data" \
-      org.label-schema.url="https://pixel.candihub.eu/" \
-      org.label-schema.vcs-ref=$VCS_REF \
-      org.label-schema.vcs-url="https://github.com/Candihub/pixel/" \
-      org.label-schema.vendor="CandiHub" \
-      org.label-schema.version=$VERSION \
-      org.label-schema.schema-version="1.0"
+    org.label-schema.name="Pixel" \
+    org.label-schema.description="Integration of smart 'omics' data" \
+    org.label-schema.url="https://pixel.candihub.eu/" \
+    org.label-schema.vcs-ref=$VCS_REF \
+    org.label-schema.vcs-url="https://github.com/Candihub/pixel/" \
+    org.label-schema.vendor="CandiHub" \
+    org.label-schema.version=$VERSION \
+    org.label-schema.schema-version="1.0"
 
 # Install Pipenv
 RUN pip install pipenv --upgrade
 
 # Add a non-privileged user for installing and running
 # the application
-RUN groupadd --gid $GID app && \
+RUN groupadd -f --gid $GID app && \
     useradd --uid $UID --gid $GID --home /app --create-home app
 
 RUN su app -c "mkdir /app/pixel"

--- a/Makefile
+++ b/Makefile
@@ -4,16 +4,11 @@ FONTS_DIR         = static/fonts
 FOUNDATION_PATH = node_modules/foundation-sites
 FA_PATH = node_modules/font-awesome
 
-# Node
-YARN_RUN = yarn
-NODEMON  = $(YARN_RUN) nodemon
-POSTCSS  = $(YARN_RUN) postcss
-SASS     = $(YARN_RUN) node-sass
-
 # Docker
 COMPOSE              = bin/compose
 COMPOSE_RUN          = $(COMPOSE) run --rm
 COMPOSE_RUN_WEB      = $(COMPOSE_RUN) web
+COMPOSE_RUN_NODE     = $(COMPOSE_RUN) node
 MANAGE               = bin/manage
 COMPOSE_TEST         = docker-compose -f docker-compose.test.yml -p pixel-test
 COMPOSE_TEST_RUN     = $(COMPOSE_TEST) run --rm

